chorebot.rb

```diff
@@ -12,27 +12,43 @@ def post_message(message)
   HTTParty.post(ENV['SLACK_WEBHOOK_URL'], body: params.to_json)
 end
 
-def assignees_for(day, candidates)
-  i1 = 2*(day.cweek*5 + day.wday) + 5
-  i2 = i1 + 1
-  member1 = candidates[i1 % candidates.length]
-  member2 = candidates[i2 % candidates.length]
-  [member1, member2]
+def assignees_for(day, candidates, number_of_chores = 3 )
+  weekday_of_year = day.cweek * 5 + day.wday
+  start_value = 5
+  seq = number_of_chores * weekday_of_year + start_value
+
+  number_of_chores.times.map { |i| candidates[(seq + i) % candidates.length] }
 end
 
-def current_trash_assignees
-  member1, member2 = assignees_for(Date.today, member_names)
-  "<@#{member1}> and <@#{member2}>"
+def assign_chores(date = Date.today)
+  member1, member2, member3  = assignees_for(date, member_names).shuffle
+  {
+    trash: "<@#{member1}> and <@#{member2}>",
+    dishes: "<@#{member3}>"
+  }
 end
 
-def morning_trash_message
-  post_message("Good morning #{current_trash_assignees}! It's your turn to take out the trash!")
+def morning_chore_message
+  current_assignees = assign_chores
+  post_message("Good morning #{current_assignees[:trash]}! It\'s your turn to take out the trash!")
+  post_message("And howdy #{current_assignees[:dishes]}. You are in charge of dishes.")
 end
 
-def afternoon_trash_message
-  post_message("Good afternoon #{current_trash_assignees}! Did you remember to take out the trash?")
+def afternoon_chore_message
+  current_assignees = assign_chores
+  post_message("#{current_assignees[:trash]}! Did you remember to take out the trash?")
+  post_message("Oh, and  #{current_assignees[:dishes]}, did you remember the dishes?")
 end
 
 def weekly_cleanup_message
-  post_message("Hey <@channel>, time to clean up the office!")
+  post_message("Hey <!group>, time to clean up the office!")
+end
+
+def weekly_snack_message
+  snack_czar = assignees_for(Date.today, member_names, 1).first
+  post_message("I hereby appoint <@#{snack_czar}> to be this week's Snack Czar. \n Forgot what to do? \n https://docs.google.com/document/d/1ZM1W3eJc4qX2-OJamy4-K4lXrPczgk-VO8hlOFdPbq0/edit")
```

Comment:
Hmm, I think the `snack_czar` position might not rotate weekly. We might need to do:
```ruby
candidates = member_names
snack_czar = candidates[Date.today.cweek % candidates.length]
```
To ensure the snack czardom is bestowed on all Vermonsters somewhat fairly.

If we want it to be a little more random (i.e. non-alphabetical), we could apply some kind of deterministic but random-seeming sorting:
```ruby
require 'digest'
candidates = member_names.sort_by { |name| Digest::SHA256.hexdigest(name) }
```